Huawei/hi6250/ANE/res/values/config.xml

```diff
@@ -96,7 +96,7 @@
             1 - Toggle theater mode setting
             2 - Brightness boost
     -->
-    <integer name="config_doublePressOnPowerBehavior">2</integer>
+    <integer name="config_doublePressOnPowerBehavior">0</integer>
```

Comment:
Uhhhhh yeah, wtf was that. @Dil3mm4 ?

Reply:
Dumbness that will be rebased along brightness values for Huawei devices.
My bad.